Branch-SDK/src/io/branch/referral/ShareLinkManager.java

```diff
@@ -31,6 +31,7 @@
     /* The custom chooser dialog for selecting an application to share the link. */
     AnimatedDialog shareDlg_;
     Branch.BranchLinkShareListener callback_;
+    Branch.BranchLinkShareListener2 callback2_;
```

Comment:
can we name these a little more distinctly now that they have different functions?